src/icon.c

```diff
@@ -63,8 +63,10 @@ static GdkPixbuf *get_pixbuf_from_file(const char *filename)
         if (is_readable_file(filename)) {
                 GError *error = NULL;
                 pixbuf = gdk_pixbuf_new_from_file(filename, &error);
-                if (!pixbuf)
+                if (!pixbuf) {
+                        LOG_M("%s", error->message);
```

Comment:
- Please use the warning log level. The error is correctable, but it's only correctable by ignoring the data.
- Please do not pipe the error message directly into the log. Give it some context. If `gdk-pixbuf` has got complete control over the message and it would give a generic error message like `Permission Denied`, you'd have only `Permission Denied`, but the user wouldn't know why and where the permission got denied. I'd suggest `LOG_W("Cannot load icon '%s' : %s", filename, error->message);`